src/main/frontend/util.cljc

```diff
@@ -633,7 +633,7 @@
 
 (defn replace-first-ignore-case
   [s old-value new-value & [escape-chars]]
-  (let [escape-chars (or escape-chars "[]{}().+*?")
+  (let [escape-chars (or escape-chars "[]{}().+*?|")
```

Comment:
The value of escape-chars is equal as above in Line 626, right? Let's reuse it by introducing a symbol here.

Reply:
Ok!